src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/UpsertResource_7.sql

```diff
@@ -100,7 +104,9 @@ CREATE PROCEDURE dbo.UpsertResource_6
     @tokenQuantityCompositeSearchParams dbo.BulkTokenQuantityCompositeSearchParamTableType_1 READONLY,
     @tokenStringCompositeSearchParams dbo.BulkTokenStringCompositeSearchParamTableType_1 READONLY,
     @tokenNumberNumberCompositeSearchParams dbo.BulkTokenNumberNumberCompositeSearchParamTableType_1 READONLY,
-    @isResourceChangeCaptureEnabled bit = 0
+    @isResourceChangeCaptureEnabled bit = 0,
+    @comparedVersion int = NULL,
+    @isResourceChange bit
```

Comment:
>@isResourceChange bit [](http://example.com/codeflow?start=4&length=21)

Please don't call this stored procedure if code finds resources equal.
Please remove this parameter.

Reply:
Updated this with latest commit